cmd/ormb/cmd/version.go

```diff
@@ -0,0 +1,36 @@
+/*
+Copyright © 2020 NAME HERE <EMAIL ADDRESS>
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
```

Comment:
I have a question.
Do we have to add `Copyright` comment in every file?   I found that there is not `Copyright` in some other file.

Reply:
In the best case, we should add the header to every file. Now we do not follow the best practice.

Reply:
OK, I know